### ISSUE TITLE
Request for installing JAGS

### ISSUE THREAD

**Issue by XinweiBuffalo** [ubccr/software-layer#303]
Could you please help me installing JAGS 4.3.2 (https://mcmc-jags.sourceforge.io/)?

I see JAGS is available on Easy Build

**Comment by tonykew** [comment]
JAGS has been built, and is going through internal testing.
I will update this request once JAGS has been published.

**Comment by tonykew** [comment]
JAGS version 4.3.2 has been published to the ccrsoft/2023.01 software release

```
$ module spider jags
----------------------------------------------------------------------------
  jags: jags/4.3.2
----------------------------------------------------------------------------
    Description:
      JAGS is Just Another Gibbs Sampler. It is a program for analysis of
      Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC)
      simulation 

    Properties:
      Mathematical libraries

    You will need to load all module(s) on any one of the lines below before the "jags/4.3.2" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
$ module load gcc/11.2.0 openmpi/4.1.1 jags/4.3.2
$ jags
Welcome to JAGS 4.3.2 on Thu May 16 13:01:51 2024
JAGS is free software and comes with ABSOLUTELY NO WARRANTY
Loading module: basemod: ok
Loading module: bugs: ok
. exit
$ 
```
